py/README.md

```diff
@@ -345,14 +373,16 @@ Here's an example of materializing a table query:
 from aou_workbench_client.swagger_client.models import ResultFilters, MaterializeCohortRequest, CohortStatus
 from aou_workbench_client.swagger_client.models import TableQuery, ColumnFilter, Operator, FieldSet, AnnotationQuery
 from aou_workbench_client.cohorts import materialize_cohort
-
-temp_filter = ResultFilters(column_filter=ColumnFilter("measurement_source_value", 
-                                                        value='Temper'))
-measure_query = TableQuery(table_name="measurement",
-                          columns=['person_id', 'measurement_id', 
-                                   'measurement_date', 
-                                   'measurement_source_value',
-                                   'value_as_number'],
+from aou_workbench_client.cdr.model import Measurement
```

Comment:
While you're in here, I copy-pasted the first full example yesterday and `pd` wasn't defined. Should probably add the import statement there.

Reply:
Done.